app/jobs/discovery_report_job.rb

```diff
@@ -1,7 +1,8 @@
 class DiscoveryReportJob < ApplicationJob
   queue_as :discovery_report
 
-  def perform(*args)
+  # @param [JobRun] job_run
+  def perform(job_run)
```

Comment:
I like specific args when they're not too big, so I like this.

I *think* we may need to add whether or not it's a smpl job, but can add that param separately, to be sure.

Reply:
I think that is (or should be) all encapsulated by the OO stuff behind the `job_run` (including the `BundleContext` object).  If we want to use the same BC for different jobs (makes sense: run discovery report, then run preassebly).  Those are separate `JobRun` objects, so any additional options unique to one job type could still be parked on that object.

Reply:
currently, it's not in JobRun or DiscoveryReport -- so does it need to be ticketed?  Can we perhaps on Monday work out all the missing work for coping with smpl style data?

Reply:
is this not captured by `BundleContext#smpl_cm_style?` (given by the `smpl_cm_style` value on the `BundleContext.content_metadata_creation` enum)?  or am i missing something?  is there a different smpl job flag that is no longer captured?

Reply:
resolved F2F in infra planning meeting.  there's also still smpl specific logic in the discovery report.